test/test_core.py

```diff
@@ -2327,11 +2327,9 @@ def test_em_js(self, args, force_c):
 
   @no_wasm2js('WASM_BIGINT is not compatible with wasm2js')
   def test_em_js_i64(self):
-    err = self.expect_fail([EMCC, '-Werror', test_file('core/test_em_js_i64.c')])
+    err = self.expect_fail([EMCC, '-sWASM=0', '-Werror', test_file('core/test_em_js_i64.c')])
```

Comment:
Why this?

Reply:
Ah, that's a typo, should be `WASM=>WASM_BIGINT`... fixing.